browser/src/docdispatcher.ts

```diff
@@ -456,6 +456,12 @@ class Dispatcher {
 			// https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollLeft
 			L.DomUtil.get('spreadsheet-tab-scroll').scrollLeft = 100000;
 		};
+		this.actionsMap['columnrowhighlight'] = function () {
+			var newState = !app.map.uiManager.getHighlightMode();
+			app.map.uiManager.setHighlightMode(newState);
+			if (newState) $('#colrowhighlight').addClass('selected');
```

Comment:
these 2 lines shouldn't be needed (adding css class)
setHighlightMode should fire commandstatechange which will inform button about state change